guides/plugins/plugins/plugin-fundamentals/using-composer-dependencies.md

```diff
@@ -0,0 +1,54 @@
+# Use Composer dependencies
```

Comment:
Nope Nope Nope. Das geht alles nicht was hier steht...

- Nimm erstmal ein anderes Package was nicht in Shopware 6 drin ist. Sonst wirst du erstmal nix merken
- Vor dem composer install im Plugin Ordner muss man alle shopware/* requires rausnehmen, sonst hat man Shopware 6 in sein Plugin Ordner
- Die nachträglich wieder einfügen
- Der Vendor Ordner des Plugins muss mitgeliefert werden
- Du musst in der Bootstrap (PluginName.php) den Vendor Ordner selber laden.
https://github.com/FriendsOfShopware/FroshPlatformTemplateMail/blob/master/src/FroshPlatformTemplateMail.php#L15-L17

Reply:
Was Shyim sagen will:
Es muss erklärt werden, dass erstmal alle requires entfernt müssen, die Shopware selbst (also platform o.Ä.) laden. Dann `composer install` ausführen, damit Composer die Packages lädt und dann nachträglich das requires wieder einfügen.